### PR TITLE
Small switch tidy-ups

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Yay, D2 catches a bug!

**Comment by nemanja-boric-sociomantic** [comment]
Finally some usage!

**Comment by gavin-norman-sociomantic** [comment]
Updated with D2 fixes.

File: src/dhtproto/client/UsageExamples.d [hunk 1]
```diff
@@ -163,7 +151,7 @@ version ( UnitTest )
                     this.log.error(this.msg_buf);
                     break;
 
-                mixin(handleInvalidCases);
+                mixin(typeof(info).handleInvalidCases);
```
Comment (nemanja-boric-sociomantic) [inline src/dhtproto/client/UsageExamples.d:154]: I think you should just use `mixin(info.handleInvalidCases)` here.

Comment (nemanja-boric-sociomantic) [reply src/dhtproto/client/UsageExamples.d:154]: Ah, no sorry, this is fine - it's just the public alias inside the type. However, you've missed one at line 624.

Comment (nemanja-boric-sociomantic) [reply src/dhtproto/client/UsageExamples.d:154]: (that's why the CI fails).

Comment (gavin-norman-sociomantic) [reply src/dhtproto/client/UsageExamples.d:154]: Updated.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Pull request merged**